### PR TITLE
feat: Add AnchorCredential type and preload contexts

### LINKED ISSUE #128: Add AnchorCredential type and preload contexts
Add AnchorCredential type and relevant contexts to anchor credential.
Add preloading of the orb contexts.

### PR DESCRIPTION
**Pull request by sandrask**
Add AnchorCredential type and relevant contexts to anchor credential.
Add preloading of the orb contexts.

Closes #128

Signed-off-by: Sandra Vrtikapa <sandra.vrtikapa@securekey.com>

### CODE REVIEW

File: pkg/anchor/builder/context.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+                "orb": "https://trustbloc.dev/ns/orb#",
+        
+                "cid": "orb:contentIdentifier"
+            }
+        }
+    }
+}`
+
+// JwsContext is jww context content.
```
Comment (troyronda) [inline pkg/anchor/builder/context.go:82]: Typo

File: pkg/anchor/builder/context.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+	// AnchorContextURI is anchor credential context URI.
+	AnchorContextURI = "https://trustbloc.github.io/did-method-orb/contexts/anchor/v1"
+
+	// JwsContextURI is jws context.
+	JwsContextURI = "https://w3id.org/jws/v1"
+)
+
+// AnchorContext is anchor context.
+const AnchorContext = `
```
Comment (troyronda) [inline pkg/anchor/builder/context.go:21]: Note we could end up with multiple versions over time.

File: pkg/anchor/builder/context.go [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                "orb": "https://trustbloc.dev/ns/orb#",
+        
+                "cid": "orb:contentIdentifier"
+            }
+        }
+    }
+}`
+
+// JwsContext is jww context content.
+const JwsContext = `
```
Comment (troyronda) [inline pkg/anchor/builder/context.go:83]: Note we could end up with multiple versions over time.

**Review by troyronda — APPROVED** [review]

**Review by bstasyszyn — APPROVED** [review]

File: pkg/anchor/builder/context.go (hunk 1 above)
Comment (sandrask) [reply pkg/anchor/builder/context.go]: fixed

File: pkg/anchor/builder/context.go (hunk 3 above)
Comment (sandrask) [reply pkg/anchor/builder/context.go]: fixed

File: pkg/anchor/builder/context.go (hunk 2 above)
Comment (sandrask) [reply pkg/anchor/builder/context.go]: fixed

**Pull request merged**